### DISCUSSION TITLE
Which grant requests should require interaction?

### DISCUSSION THREAD

**Discussion by sabineschaller** [interledger/rafiki#838] [Q&A]
Currently, grant requests that require interaction are for
* quotes
* outgoing payments.

Incoming payment grant requests don't require interaction.

Should quote grant requests also not require interaction?

**Comment by adrianhopebailie** [comment]
This should be decision of the AS but I think we can provide guidance. 

I don't think you should need user consent to get a quote but we should then make sure it's possible to get a quote without revealing private info (like a balance)
